tmt/steps/provision/testcloud.py

```diff
@@ -638,6 +638,9 @@ def go(self) -> None:
             for key in TestcloudGuestData.keys()
             })
 
+        # rhcos does not allow root login, set default user core
+        if ("rhcos" in data.image.lower() and not self.opt('user')):
```

Comment:
Nitpicking: wrapping the whole condition with `(` & `)` is not needed in Python :)